### PR TITLE
added `<Table.Subtoolbar>` component

### PR DESCRIPTION
**Pull request by dorduch**
### 🔦 Summary

...

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [ ] Visual test

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://www.wix.com/pages/wix-style-react.pr_5624/

File: src/Table/docs/examples/TablePageExample.js [hunk 1]
```diff
@@ -56,6 +56,11 @@ class TablePageExample extends React.Component {
                       : this._renderActionsToolbar({ ...selectionContext })
                   }
                 </Table.ToolbarContainer>
+                <Table.SubToolbar>
```
Comment (shlomitc) [inline src/Table/docs/examples/TablePageExample.js:59]: I don't think this is needed here

File: src/Table/index.d.ts [hunk 2]
```diff
@@ -35,6 +36,7 @@ export default class Table<
 }
 
 declare const ToolbarContainer: React.SFC;
+declare const SubToolbar: React.FunctionComponent<{ dataHook?: string }>;
```
Comment (shlomitc) [inline src/Table/index.d.ts:39]: maybe `SFC`, same as the others?

Comment (dorduch) [reply src/Table/index.d.ts:39]: `SFC` is deprecated in favour of `FunctionComponent`

File: src/Table/docs/examples/TablePageExample.js (hunk 1 above)
Comment (dorduch) [reply src/Table/docs/examples/TablePageExample.js:59]: np, removing

Comment (dorduch) [reply src/Table/docs/examples/TablePageExample.js:59]: didn't we want to show an example of how to use it with sticky?

Comment (shlomitc) [reply src/Table/docs/examples/TablePageExample.js:59]: It is already displayed in the new example you added, no?

Comment (dorduch) [reply src/Table/docs/examples/TablePageExample.js:59]: the example is without sticky

Comment (shlomitc) [reply src/Table/docs/examples/TablePageExample.js:59]: So change it to be sticky. In 90% of the cases, table in a page should have a sticky header

**Comment by dorduch** [comment]
@milkyfruit  how can I know if `TitleBar` is visible?

**Comment by milkyfruit** [comment]
@dorduch ok, I checked how table is implemented. Let's just remove bottom border from `SubToolbar`, if users need it they will be able to add  `<Divider/>` component manually.

**Pull request merged**